### ISSUE TITLE
Make separate libraries for different softwares

### ISSUE THREAD

**Issue by sofian** [SofaPirate/Massenger#2]
Currently, support for Max, Processing, TouchDesigner, etc. are provided in a separate folder in each library. However, I think what would be nice is to make the libraries available under different projects (repositories) (eg. processing-AsciiMassage) so that for example they can be installed more easily from within each of these platforms.

However I would also recommend writing an online documentation that would reflect this and point to the right resources. See for example how Libossia (https://ossia.github.io/) does it for documentation on multi-platform (not saying they are the best example, actually, but it's just to show the idea that Massage could be portrayed as a multi-platform / multi-language standard rather than just an Arduino library).

**Comment by thomasfredericks** [comment]
Yes, that would be ideal, but I do not have the time to make all that documentation. What I could do is split the AsciiMassage implementations for Processing and friends into separate repositories. But a new problem arises, where the examples are split (one half in the AsciiMassage Arduino repository and the other in the Processing repository).

**Comment by sofian** [comment]
Maybe it's okay to keep it that way for now. At my University I could have a student work on this in the Fall if you could help give them some supervision. We can find someone with basic knowledge of code who is also good at writing documentation.
